### PR TITLE
NO-ISSUE: Set workers hostname in 'none' platform according to their network

### PR DESCRIPTION
**Pull request by danmanor**

### CODE REVIEW

File: terraform_files/none/main.tf [hunk 1]
```diff
@@ -113,7 +113,7 @@ module "workers" {
   source = "../baremetal_host"
   count  = var.worker_count
 
-  name           = "${var.cluster_name}-worker-${count.index}"
+  name           = count.index % 2 == 0 ? "${var.cluster_name}-worker-${count.index}" : "${var.cluster_name}-worker-secondary-${count.index}"
```
Comment (eifrach) [inline terraform_files/none/main.tf:116]: So if you have 2 workers the first worker will be worker-secondary-1 and the second will be worker-2. Why?

**Review by eifrach — CHANGES_REQUESTED** [review]

File: terraform_files/none/main.tf (hunk 1 above)
Comment (eifrach) [reply terraform_files/none/main.tf:116]: btw this is the names from the CI run's 
```
"name": "test-infra-cluster-8b3106c1-worker-0",
"name": "test-infra-cluster-8b3106c1-worker-secondary-1",
```

Comment (danmanor) [reply terraform_files/none/main.tf:116]: Yes, the idea is to label the worker based on its network for easier debugging. Notice that this is what we do in maters already

**Comment by danmanor** [comment]
/override ci/prow/e2e-metal-assisted-kube-api-late-binding-sno-4-18

File: terraform_files/none/main.tf (hunk 1 above)
Comment (eifrach) [reply terraform_files/none/main.tf:116]: to this set the name if the host number is even or not.  
so I'm not sure if this is the result you wanted here

Comment (danmanor) [reply terraform_files/none/main.tf:116]: Yes, to split the network hosts half-half. I just copied the code from masters

**Pull request merged**